tests/test_cli.py

```diff
@@ -41,7 +41,7 @@ def test_cli_tox_ini_not_file(tmp_path, capsys):
     ],
 )
 @pytest.mark.skipif(sys.platform == "win32", reason="On Windows files cannot be read only, only folders")
-def test_cli_tox_ini_permission_fail(tmp_path, capsys, flag, error):
+def test_cli_tox_ini_permission_fail(tmp_path, capsys, flag, error):  # pragma: no cover
```

Comment:
Why do we need this no cover?

Reply:
on windows it gives an error during running tox. IMHO the test code shouldn't be part of the coverage, but that's just how I usually do it.

Reply:
use platform specific cover pragma per https://pypi.org/project/covdefaults/

Reply:
> IMHO the test code shouldn't be part of the coverage, but that's just how I usually do it.

The maintainer of coveragepy disgarees and I agree with him https://nedbatchelder.com/blog/202008/you_should_include_your_tests_in_coverage.html